### DISCUSSION TITLE
Delayed commands

### DISCUSSION THREAD

**Discussion by Kesims** [Zrips/CMI#4960] [Feature Requests]
A simple command, that would delay its processing. Eg. /cmi delayed 1m say hello - That would say hello one minute after using the command. Would that be possible? Thanks

**Comment by hamcoremusic** [comment]
You could make a custom alias with a specialized command to do this. Check out the [custom alias](https://www.zrips.net/cmi/commands/custom-alias/) and [specialized commands](https://www.zrips.net/cmi/commands/specialized/) pages on their website!

**Reply by Kesims** [reply]
Uh, how can I add some args from alias to the original command? Cuz i need to run "cr give to [PlayerName] VoteKey" - but it is executed from console

**Reply by hamcoremusic** [reply]
You can use `$<number of arg>` like $1 is the first argument, so you would make an alias like you did and the command would be `asConsole! cr give to $1 VoteKey`

For longer than one word arguments, you can put a `-` like `$1-` which will then take the first STRING instead of just one value.

**Reply by hamcoremusic** [reply]
This is outlined on the top of the custom alias page I linked above.

**Reply by Kesims** [reply]
Ohh, I see, I'm just ... stupid. Thank you very much!
